src/test/java/org/opensearch/ad/e2e/DetectionResultEvalutationIT.java

```diff
@@ -592,26 +593,14 @@ private void indexTrainData(String datasetName, List<JsonObject> data, int train
                 throw new RuntimeException(e);
             }
         });
-        Thread.sleep(1_000);
+        Thread.sleep(3_000);
     }
 
     public void testRestartHCADDetector() throws Exception {
         // TODO: this test case will run for a much longer time and timeout with security enabled
         if (!isHttps()) {
-            int maxRetries = 3;
-            int i = 0;
-            for (; i < maxRetries; i++) {
-                try {
-                    disableResourceNotFoundFaultTolerence();
-                    verifyRestart("synthetic", 1, 8);
-                    break;
-                } catch (Throwable throwable) {
-                    LOG.info("Retry restart test case", throwable);
-                    cleanUpCluster();
-                    wipeAllODFEIndices();
-                }
-            }
-            assertTrue("failed all retries", i < maxRetries);
+            disableResourceNotFoundFaultTolerence();
+            verifyRestart("synthetic", 1, 8);
```

Comment:
Previous code had `cleanUpCluster()` and `wipeAllODFEIndices()` if there was some failure. Will the new retry logic handle this for us?

Reply:
good catch. I don't believe the retry plugin does these operations, maybe we should add these operations to the end of the test?

Reply:
What about removing the for loop still, but adding back the try/catch? That way you can keep the same functionality, where if there's some exception it will run `cleanUpCluster()`/`wipeAllODFEIndices()`. Can also fail the test in catch block, else pass it.

Reply:
This makes sense to me and I'll add a `fail()` in catch block as you mention, only caveat is that there could be cases where `verifyRestart` fails and no exception thrown so it won't run clean and wipe methods. However, i think when this test is flaky it might always throw the exception and not just fail on assertion before. @kaituo can you confirm? I